support-frontend/assets/components/headers/header/header.tsx

```diff
@@ -1,22 +1,18 @@
 // ----- Imports ----- //
-import type { Node } from 'react';
 import { Component } from 'react';
 import SvgGuardianLogo from 'components/svgs/guardianLogo';
-import 'helpers/types/option';
-import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
-import 'helpers/internationalisation/countryGroup';
 import { getGlobal } from 'helpers/globalsAndSwitches/globals';
+import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
 import type { ElementResizer } from 'helpers/polyfills/layout';
 import { onElementResize } from 'helpers/polyfills/layout';
-import type { Option } from 'helpers/types/option';
 import { classNameWithModifiers } from 'helpers/utilities/utilities';
 import Links from '../links/links';
 import MobileMenuToggler from './mobileMenuToggler';
 import Padlock from './padlock.svg';
 import './header.scss';
 
 export type PropTypes = {
-	utility: Option<Node>;
+	utility: JSX.Element | null;
```

Comment:
```suggestion
	utility?: JSX.Element;
```
If possible it'd be really good to move towards optional properties- not sure if this would involve messing around with another file though.

Reply:
Yeah agreed - I'll have a go...

Reply:
Nice! No knock-ons so super simple to fix